### ISSUE TITLE
Run.Bat won't execute

### ISSUE THREAD

**Issue by billnltc** [derekhe/msfs2020-google-map#166]
Hi... I have downloaded the latest version (2.3.4), extracted to my Desktop, and have selected to run the run.bat file as Administrator, however, it will not run.  The path for the python.exe cannot be located.  See screenshot.

![Screenshot 2021-12-15 021225](https://user-images.githubusercontent.com/96168077/146140795-ce489399-bbeb-43cd-bf09-b1864f6bba40.png)

I previously had the program installed and it worked perfectly, but I had to restore my system to a prior date and, as a result, need to re-install now.

**Desktop (please complete the following information):**
 - OS: Windows 11
 - MOD Version [e.g. 22]: Not sure what this is
 - MSFS Version, MS Store or Stream:  MSFS 2020 Basic Version from MS Store

Thank you very much for your time and guidance.

**Comment by matead92** [comment]
Install Python and fix your execution argument path in RUN.BAT. From your screenshot, you're trying to change to a directory on the desktop called "Python39" and then you're trying to run the script (which won't work because it doesn't exist).

**Comment by Rockplough** [comment]
Yeah this is annoying. Editing the batch file to properly point to my python directory seems to have fixed it. In my case:

C:\Users\final\anaconda3\python.exe src\app.py
